apps/aehttpserver/web/controllers/tx_controller.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Aehttpserver.TxController do
+  use Aehttpserver.Web, :controller
+  alias Aecore.Txs.Pool.Worker, as: Pool
+  alias Aecore.Utils.Serialization, as: Serialization
+  alias Aecore.Structures.TxData
+
+  def show(conn, params) do
+    account_bin =
+      params["account"]
+      |> Base.decode16!()
+    user_txs = Pool.get_txs_for_address(account_bin)
+    case user_txs do
+      [] -> json(conn, [])
+      _ ->
+        case params["include_proof"]  do
+          "true" ->
+            blocks_for_user_txs =
```

Comment:
We should move all of the proof adding code to a function so we can test it. (and create such tests)